ImportUsers/UserDetails.cs

```diff
@@ -3,36 +3,50 @@
 namespace Geotab.SDK.ImportUsers
 {
     /// <summary>
-    /// Object with the users details
+    /// Object representing user details.
     /// </summary>
-    class UserDetails
+    /// <remarks>
+    /// Fields include user information such as email, password, data access, security clearance, first name, and last name.
+    /// </remarks>
+    /// <remarks>
+    /// Initializes a new instance of the <see cref="UserDetails"/> class.
+    /// </remarks>
+    /// <param name="user">The user.</param>
+    /// <param name="password">The password.</param>
+    /// <param name="dataAccess">The data access.</param>
+    /// <param name="securityClearance">The security clearance.</param>
+    /// <param name="firstName">The first name.</param>
+    /// <param name="lastName">The last name.</param>
+    class UserDetails(User user, string password, string dataAccess, string securityClearance, string firstName, string lastName)
     {
         /// <summary>
-        /// The organization node names
+        /// The user.
         /// </summary>
-        public readonly string OrganizationNodeNames;
+        public readonly User UserNode = user;
 
         /// <summary>
-        /// The security node name
+        /// The password.
         /// </summary>
-        public readonly string SecurityNodeName;
+        public readonly string PasswordNode = password;
 
         /// <summary>
-        /// The user
+        /// The data access.
         /// </summary>
-        public readonly User User;
+        public readonly string DataAccessNode = dataAccess;
```

Comment:
To  avoid confusion with the new UI, let's change this to be `GroupNode` instead